src/sentry/migrations/0655_add_status_column_for_alert_rule_trigger_action.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 5.0.2 on 2024-02-24 01:31
+
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
```

Comment:
we're adding an index here because the manager we add will want to filter on `active` status only. Might have to make this a dangerous migration now for the table?

Reply:
Won't any queries we make be on other values though, like `alert_rule_trigger_id`? The index won't do a lot i those cases, I'm not totally sure we need it.

The table has 200k rows, I *think* it will run in time but am not sure.

Reply:
Because we're going to add a manager for this table what will always add `.filter(status=ACTIVE)`, I was thinking the index would be beneficial! Just so it's not a hinderance and slow down the line, but you're right that our current flows don't really heavily leverage this table. I can leave it out!

Reply:
removed the index, we can add on when it becomes a performance issue